### ISSUE TITLE
Random FanArt - Verify FanArt Exists First

### ISSUE THREAD

**Issue by MickaelMartinNevot** [mikesilvo164/Aeon-Nox-SiLVO#167]
Random fanart, like "Kodi random movie fanart" seems to be selected 100% randomly. It does not take into account movies that do not have any fanart.
So many times it randomly selects movies that have no fanart so the user sees a black screen instead of an expected random fanart.

As expected behavior, random fanart is still selected randomly, but now it first verifies the movie and/or TV show selected at random, indeed has fanart to use.
Basically if randomly selected movie and/or TV show selected has no fanart then we have to pick another random movie and/or TV show until we find one that does.

I found in formal 5.8.2 Aeon Nox BigNoid version "- Fix black background on Home screen for uncached fanart.": it seems to be the same issue, but it should be fixed now if I understand correctly.

**Comment by mikesilvo164** [comment]
There is no "smarts" to the Kodi random fanart. If there are items in your library that do not have fanart and you do not want to see any black backgrounds then you can install script.grab.fanart and select the random fanart that does not have "kodi" in front of the type. 

![image](https://i.imgur.com/pX84yQD.png)

OG Aeon Nox 5 also used script.grab.fanart.

**Comment by MickaelMartinNevot** [comment]
Only manual install of Grab Fanart seems to work nowdays (on Matrix), but it works well, and also offer a very welcome full random of every kind of media!

Thank you very much!
